Source/Core/Common/StringUtil.cpp

```diff
@@ -450,6 +467,11 @@ std::string SHIFTJISToUTF8(const std::string& input)
   return UTF16ToUTF8(CPToUTF16(932, input));
 }
 
+std::string UTF8ToSHIFTJIS(const std::string& input)
+{
+  return UTF16ToCP(932, UTF8ToUTF16(input));
```

Comment:
Can this number be less magic? For example by declaring `constexpr u32 SHIFT_JIS_CODE_PAGE = 932;` somewhere.